Dockerfile.builder

```diff
@@ -2,14 +2,13 @@ FROM centos:7
 LABEL maintainer "Devtools <devtools@redhat.com>"
 LABEL author "Konrad Kleine <kkleine@redhat.com>"
 ENV LANG=en_US.utf8
-ARG USE_GO_VERSION_FROM_WEBSITE=1
 
 # Some packages might seem weird but they are required by the RVM installer.
 RUN yum install epel-release -y \
-    && yum install --enablerepo=centosplus install -y --quiet \
+    && yum --enablerepo=centosplus --enablerepo=epel install -y --quiet \
       findutils \
       git \
-      $(test "$USE_GO_VERSION_FROM_WEBSITE" != 1 && echo "golang") \
+      golang \
```

Comment:
I highly object against this change. I explicitly want to be able to use Go from the website in order to test out new features in builds like the coverage for example. If you find that CentOS has ditched Golang as it appears, then please fix that problem and don't just remove the ability to use Go from the website.

Reply:
>I highly object against this change. I explicitly want to be able to use Go from the website in order to test out new features in builds like the coverage for example. 

We can still test things via a PR, can't we? Even if we keep the flag, you'd still have to send a new PR if you want to test something :)


>If you find that CentOS has ditched Golang as it appears, then please fix that problem and don't just remove the ability to use Go from the website.

Using golang from epel is exactly what we want. I wasn't part of the discussion where it was decided to use golang from epel and not website but it looks like most of our services have already migrated https://github.com/openshiftio/openshift.io/issues/4618

Reply:
By testing I mean: Permanently use a feature (not only in one PR).

I'm not suggesting to not use Go from epel. I just want you to not remove the feature to be able to use Go from the website.

Reply:
@kwk okay. That makes sense. I'll undo the removal of `from-website` flag.